SPECS/kernel/config

```diff
@@ -7327,6 +7328,7 @@ CONFIG_LOCK_DOWN_KERNEL_FORCE_NONE=y
 # CONFIG_LOCK_DOWN_KERNEL_FORCE_INTEGRITY is not set
 # CONFIG_LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY is not set
 CONFIG_SECURITY_LANDLOCK=y
+# CONFIG_SECURITY_IPE is not set
```

Comment:
@rlmenge Let's enable the IPE capability in an upcoming update. The feature will stay inert until a policy is provided which is acceptable.